lib/autoyast.pm

```diff
@@ -843,6 +840,11 @@ sub prepare_ay_file {
     $profile = expand_version($profile);
     $profile = adjust_network_conf($profile);
     $profile = expand_variables($profile);
+
+    if (check_var('IPXE', '1')) {
+        $path = get_required_var('SUT_IP') . $path;
```

Comment:
The code in original place cannot work, may I know why we need connect get_required_var('SUT_IP') and $path? Thanks!

Reply:
Ask @frankenmichl who wrote the original code (before it was broken by refactoring). I assume that there was some concern about clobbering AutoYaST profiles generated for different IPMI machines. This is the correct way to modify the path.

Reply:
Thanks! I tried to remove the code block and verified with different IPMI machine parallel, the 3 autoyast path are unified for each machine, no conflict occurred, here is an example about running 3 machines parallel: 
http://10.67.134.70:20453/s6_JMKvZYPjJjwKb/files/autoyast_hanaperf/mu.xml
http://10.67.134.70:20463/9WepUQzXV6Hk3nCc/files/autoyast_hanaperf/mu.xml
http://10.67.134.70:20443/qz4FZiEf4FAGWmRP/files/autoyast_hanaperf/mu.xml

Reply:
Honestly, it has been a while, so I don't remember the details here. But I am sure it had to do with different machines of the same WORKER_CLASS.

Reply:
The autoyast profile failed to be uploaded to the job space on worker for virtualization tests with `$path = get_required_var('SUT_IP') . $path;`. I planed to removed this statement from my PR(https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17400), but since you have more concers about it,  I will remove it for virt tests only.

Reply:
Autoyast profile download fails because the path is modified in the wrong place and the download then uses the unmodified path. The fix above will work everywhere.

Reply:
Great. I'll revert my change to this path in my PR#17400.